ska_conda/pkg_defs/mica/meta.yaml

```diff
@@ -0,0 +1,61 @@
+package:
+  name: mica
+  version:  {{ GIT_DESCRIBE_TAG }}
+
+build:
+  noarch: python
+  script_env:
+    - SKA_TOP_SRC_DIR
+
+source:
+  path: {{ SKA_TOP_SRC_DIR }}/mica
+
+
+# the build and runtime requirements. Dependencies of these requirements
+# are included automatically.
+requirements:
+  # Packages required to build the package. python and numpy must be
+  # listed explicitly if they are required.
+  build:
+    - python
+    - setuptools
+    - testr
+  # Packages required to run the package. These are the dependencies that
+  # will be installed automatically whenever the package is installed.
+  run:
+    - python
+    - six
+    - setuptools
+    - matplotlib
+    - numpy
+    - pytest
```

Comment:
In general seems this should be `testr`, which is directly imported in the package and then requires `pytest`.

Reply:
It looks like the mica test packages import pytest directly so I think it is fine to have them both (testr and pytest) even if you end up with redundancy in the dependency tree.